cypress/integration/pokemon_details_spec.js

```diff
@@ -0,0 +1,130 @@
+// to start the test run npx cypress open into your terminal
+describe('Pokemon Details page user flow', () => {
+    beforeEach(() => {
+        cy.onLoad()
+    });
+// 
+    it('Should contain a Header (the Navigation bar) with a How to and a Image ', () => {
+        cy.get('.header')
+          .should('be.visible')
+          .get('.header-image')
+          .should('be.visible')
+          .get('.how-to')
+          .contains('How To')
+
+        //   we need to add test for the animations
+    });
+
+    it('Should be able to click on How to and go to the How To page', () => {
+        cy.get('a')
+          .click()
+          .get('.all-pokemon')
+          .should('not.exist')
+          .get('.help-page')
+          .should('be.visible')
+    });
+
+    it('Should be able to click on the x button and get back to the home page', () => {
+        cy.get('a')
+          .click()
+          .get('.x-icon')
+          .click()
+          .get('.help-page')
+          .should('not.exist')
+          .get('.all-pokemon')
+          .should('be.visible')
+    });
+
+    it('Should have a search field', () => {
+        cy.get('form')
+          .should('be.visible')     
+    });
```

Comment:
I wonder if there is any other way to make this a little more robust? Maybe we can see if we can test the placeholder text as well? 🧐

Reply:
This looks great. Matt, good idea too!